docs/reference/changelog.md

```diff
@@ -4,13 +4,63 @@ title: Changelog
 sidebar_label: Changelog
 ---
 
+## Darklang Release 4 (August 1, 2022)
+
+### Standard library improvements
+
+- Many standard library functions now have better descriptions, including much
+  better formatting ([#4273](https://github.com/darklang/dark/pull/4273)).
+  ![Better formatting](/img/changelog/pretty-descriptions.png)
+
+- Add new version of `Result::fromOption`, which allows non-string errors
+  ([#4272](https://github.com/darklang/dark/pull/4272)).
+
+- Add new version of `Bytes::base64Decode`, which fails properly on invalid
+  input ([#4287](https://github.com/darklang/dark/pull/4287)).
+
+- Bug fix: In the editor, Dates were incorrectly offset based on your timezone
+  ([#4260](https://github.com/darklang/dark/pull/4260)).
+
+- Improved handling of 64-bit ints in editor
+  ([#4257](https://github.com/darklang/dark/pull/4257)).
+
+### Contributor improvements
+
+- Allow contributors with ARM64 Macs to build Dark codebase
+  ([#4237](https://github.com/darklang/dark/pull/4237)).
+
+### Internal improvements
+
+- We are in the process of rewriting the backend of the editor on top of Dark,
```

Comment:
"on top of" feels a bit off to me - "with" would be fine, but also probably fine as-is.